app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/auth/AuthNavigation.kt

```diff
@@ -181,6 +183,16 @@ fun NavGraphBuilder.authGraph(
         masterPasswordGeneratorDestination(
             onNavigateBack = { navController.popBackStack() },
             onNavigateToPreventLockout = { navController.navigateToPreventAccountLockout() },
+            onNavigateBackWithPassword = {
+                navController.popBackStack(
+                    route = MASTER_PASSWORD_GUIDANCE,
```

Comment:
Is this going back twice?

Reply:
yes, it pops the intermediary screen off the stack too

Reply:
Could we create a helper extension called `NavController.popUpToCompleteRegistation()` and add that to `CompleteRegistrationNavigation`

```kotlin
fun NavController.popUpToCompleteRegistation() {
    navController.popBackStack(route = COMPLETE_REGISTRATION_ROUTE, inclusive = false)
}
```

The idea is that we do not want to expose these stringly routes.